src/nav/new-relic-solutions.yml

```diff
@@ -149,7 +149,37 @@ pages:
   - title: Build on New Relic UI
     pages:
       - title: New Relic CLI
-        path: /docs/new-relic-solutions/tutorials/new-relic-cli
+        path: /docs/new-relic-solutions/tutorials/new-relic-cli    
+      - title: New Relic One CLI commands
```

Comment:
```suggestion
      - title: New Relic CLI commands
```

Reply:
@nbaenam "New Relic One" isn't really a name we use anymore. Let's drop the "One" where we see it